### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by agnessnowplow**
Two minor changes, as discussed earlier. Let me know if there is anything else I can add during this. Thanks.

### CODE REVIEW

File: content/_index.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Here you will learn to:
 
 #### What you will learn
 
-In approximately 2 working days (~13 working hours) you can achieve the following:
+In approximately 2 working days (~8 working hours) you can achieve the following:
```
Comment (paulboocock) [inline content/_index.md:31]: Isn't 8 hours 1 working day?

I wondering if we leave the gantt with some estimates in it, but then change this to "In a couple of days..."

Comment (agnessnowplow) [reply content/_index.md:31]: Good catch! It's easier if I just shorten it to `In approximately 8 working hours`. Then the two will be aligned.

**Review by Jack-Keene — APPROVED** [review]
LGTM 👍🏻

File: content/_index.md (hunk 1 above)
Comment (paulboocock) [reply content/_index.md]: That works for me!

**Pull request merged**